### PR TITLE
Shortened status messages

### PR DESCRIPTION
**Pull request by zc-devs**
```
WOODPECKER_STATUS_CONTEXT: ci
```

new-status

### CODE REVIEW

File: server/forge/common/status.go [hunk 1]
```diff
@@ -58,21 +59,21 @@ func GetPipelineStatusContext(repo *model.Repo, pipeline *model.Pipeline, workfl
 func GetPipelineStatusDescription(status model.StatusValue) string {
 	switch status {
 	case model.StatusPending:
-		return "Pipeline is pending"
```
Comment (6543) [inline server/forge/common/status.go:61]: this is about the description please revert it back!

if it is used somewhere, and that is to long, the StatusValue should be used directly

File: server/forge/common/status.go [hunk 2]
```diff
@@ -38,12 +38,13 @@ func GetPipelineStatusContext(repo *model.Repo, pipeline *model.Pipeline, workfl
...
-		"context":  server.Config.Server.StatusContext,
-		"event":    event,
-		"workflow": workflow.Name,
-		"owner":    repo.Owner,
-		"repo":     repo.Name,
-		"axis_id":  workflow.AxisID,
+		"context":        server.Config.Server.StatusContext,
+		"event":          event,
+		"workflow":       workflow.Name,
+		"workflow_count": len(pipeline.Workflows),
```
Comment (6543) [inline server/forge/common/status.go:44]: this is the parsed pipeline ... we must make sure that even if workflows are filtered out later, they still count

File: server/forge/common/status.go (hunk 1 above)
Comment (zc-devs) [reply server/forge/common/status.go:61]: What else could be here?
[Screenshot 2024-08-31-1](https://github.com/user-attachments/assets/c097de93-6f2a-4c0f-878d-070168a8a4d5)
[Screenshot 2024-08-31-2](https://github.com/user-attachments/assets/36790461-33b7-4b68-911e-eaab6a4f31f8)
[Screenshot 2024-08-31-3](https://github.com/user-attachments/assets/72d97e09-163b-4ff5-ad2f-aa41fb1199c0)

File: server/forge/common/status.go (hunk 2 above)
Comment (zc-devs) [reply server/forge/common/status.go:44]: If this is the parsed pipeline and if workflows are filtered out _later_, then they still count here, no?

File: server/forge/common/status.go (hunk 1 above)
Comment (zc-devs) [reply server/forge/common/status.go:61]: Pipeline or workflow, BTW? :)

Comment (6543) [reply server/forge/common/status.go:61]: ah well yes it should be renamed to workflow :D

File: server/forge/common/status.go (hunk 2 above)
Comment (6543) [reply server/forge/common/status.go:44]: i dont remember that's what to find out ...

Comment (zc-devs) [reply server/forge/common/status.go:44]: Is there a simple solution to get that info?

Comment (zc-devs) [reply server/forge/common/status.go:44]: I think it's not worth it

**Comment by zc-devs** [comment]
I think, there is no much value without shortened status (the first requested change) and it's not worth to go further with the workflow count (second requested change).
So, closing it for now.

**Pull request closed without merging**